net-misc/uftp/uftp-4.10.1.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit toolchain-funcs
+
+DESCRIPTION="Encrypted UDP based FTP with multicast"
+HOMEPAGE="http://uftp-multicast.sourceforge.net/"
+SRC_URI="mirror://sourceforge/${PN}-multicast/${P}.tar.gz"
+
+LICENSE="GPL-3"
```

Comment:
The sourceforge page says it uses "other" license along with GPL-3, could you dig what it may be in the source files?